client/landing/jetpack-cloud/sections/scan/main.tsx

```diff
@@ -29,7 +30,46 @@ import contactSupportUrl from 'landing/jetpack-cloud/lib/contact-support-url';
  */
 import './style.scss';
 
-class ScanPage extends Component {
+interface Props {
+	site: object | null;
+	siteSlug: string | null;
+	scanState: Scan | null;
+	lastScanTimestamp: number;
+	nextScanTimestamp: number;
+	moment: Function;
+}
+
+class ScanPage extends Component< Props > {
+	// @todo: understand when this state can happen
+	// It seems, the user won't ever get to this place because it they don't have
+	// Scan they won't get this far in the app.
+	renderUnavailable() {
```

Comment:
There are a number of reasons to end up in this state. 
I think we should try to address it in a different PR with wording for all the different state. 

Unavailable could be because the site is a part of a multi site setup. 
It could be because the site is multisite for example.

Doesn't have the right plan / product, credentials. 

Here are some of the states.
<img width="1091" alt="Screen Shot 2020-04-24 at 1 04 31 PM" src="https://user-images.githubusercontent.com/115071/80206163-2d01d800-862c-11ea-92e4-bd7c0506139f.png">

Reply:
TIL. I agree 100%.